recipes/s3cmd/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "s3cmd" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6d7a3a49a12048a6c8e5fbb5ef42a83101e2fc69f16013d292b7f37ecfc574a0
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - python-dateutil
+    - python-magic
+
+test:
+  commands:
+    - s3cmd --help
+
+about:
+  home: https://s3tools.org/s3cmd
+  license: GPL v2
```

Comment:
I'm trying to standardize this based on the SPDX strings to make it easier to read them in the future. This one would be GPL-2.0 according to https://spdx.org/licenses/GPL-2.0.html